SingleCrystal/anvred3.py

```diff
@@ -115,8 +116,12 @@
 from spectrumCalc import *
 from absor_sphere import *
 
-from Tkinter import *
-import tkSimpleDialog
+if sys.version_info > (3,):

```

Comment:
Just a quick question. Does this not say greater than 3 and not equal or greater to 3?

Reply:
Tested and confirmed that this is actually fine due to the presence of the minor and patch fields. :+1: